tests/run-centos-ci.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# setting up virtual environment
+yum -y install epel-release
+virtualenv --system-site-packaes env
```

Comment:
Both this and the next step can only be done after the python-virtualenv package is installed.

Typo - `--system-site-packages`

Reply:
added installation line.